addons/SimpleTODO/TODO.gd

```diff
@@ -1,16 +1,31 @@
 tool
-extends VBoxContainer
+extends Control
```

Comment:
![image](https://user-images.githubusercontent.com/2223172/177056980-bb2217d7-1d49-45c4-b893-e8d48a5ac14d.png)
The VBoxContainer here could become the root probably.

Reply:
When your dragging a column or item, I change the parent to be the main TODO node which is a regular control node.
From what I remember it behaved strangely when dragging items around when it was parented to the VBoxContainer.

Reply:
Ah that could make sense.